### PR TITLE
Replace herd with relgroup on more pages

### PR DESCRIPTION
**Pull request by hasegeli**

### CODE REVIEW

File: Bucardo/cli/add_relgroup.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title: Bucardo add relgroup
+---
+
+The **add relgroup** command is used to create a new [Bucardo relogroup](/Bucardo/object_types/relgroup)
```
Comment (jonjensen) [inline Bucardo/cli/add_relgroup.md]: typo in "relogroup"

**Review by jonjensen — COMMENTED** [review]
@hasegeli Thanks for this!

I noted one typo inline.

(Thinking out loud:) I'm wondering about how you removed the `bucardo.` schema designation, but I guess it makes sense by removing clutter. And it seems fine to say once "this lives in the `bucardo` schema" and not repeat it everywhere, especially if a different schema name could be used.

File: Bucardo/cli/add_relgroup.md (hunk 1 above)
Comment (hasegeli) [reply Bucardo/cli/add_relgroup.md]: Fixed

**Comment by hasegeli** [comment]
I removed `bucardo.` from the URLs when I moved them under `schema/`.  This merge request removes `Bucardo.` from the titles.  All of those pages still has an header like "Table: bucardo.\<table\>".

**Pull request merged**